### PR TITLE
Test/97/unit test app text area

### LINKED ISSUE #97: (SP: 1) Write unit test for "AppTextArea" component
### Component unit test
Unit test for "**AppTextArea**" component
Scenaries descriptions:
- [ ] If title was passed via props it should render title
- [ ]  If title wasn't passed via props it shouldn't be rendered title

[Link to component](https://github.com/Space2Study-UA-1156/Client-01/blob/develop/src/components/app-text-area/AppTextArea.jsx)

Current coverage:
image

### PR DESCRIPTION
**Pull request by OlhaOvsiienko**
Here are the tests for AppTextArea Component.
image

Thanks!

### CODE REVIEW

File: src/containers/guest-home-page/how-it-works/HowItWorks.jsx [hunk 1]
```diff
@@ -18,10 +18,10 @@ import { styles } from '~/containers/guest-home-page/how-it-works/HowItWorks.sty
 const HowItWorks = () => {
```
Comment (holus-bolus) [inline src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: You created the unit tests for the AppTextArea, but the HowItWorks component would be overwritten by your commit
image


I think you should adjust your commits

**Comment by khytrykv** [comment]
You should create a new branch from the develop branch, and add the file with the tests and create a new pull request.

**Review by khytrykv — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

File: src/containers/guest-home-page/how-it-works/HowItWorks.jsx (hunk 1 above)
Comment (OlhaOvsiienko) [reply src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: Hello, Oleksii!
I Irejected unnecessary commits. Can you watch?
Thank you!

Comment (holus-bolus) [reply src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: Great job

**Review by holus-bolus — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**